webapp/app.py

```diff
@@ -0,0 +1,48 @@
+import flask
+from werkzeug.contrib.fixers import ProxyFix
+from werkzeug.debug import DebuggedApplication
+
+import talisker.flask
+from webapp.handlers import add_headers, clear_trailing_slash
+from webapp.landscape.views import landscape
+
+
+def create_app(testing=False):
+    app = flask.Flask(
+        __name__, template_folder='../templates', static_folder='../static')
+
+    app.testing = testing
+
+    app.wsgi_app = ProxyFix(app.wsgi_app)
+    if app.debug:
+        app.wsgi_app = DebuggedApplication(app.wsgi_app)
```

Comment:
you removed a line `talisker.flask.register(app)` just above

Reply:
sorry it shoulnd't be there but after it

```
    if not app.testing:
        talisker.flask.register(app)
```

Reply:
so like:
```
    if app.debug:
        app.wsgi_app = DebuggedApplication(app.wsgi_app)

    if not app.testing:
        talisker.flask.register(app)
```
?

Reply:
perfect